about.html

```diff
@@ -0,0 +1,232 @@
+          <li class="single-cont-author">
+            <a href="#" class="link-nav"><i class="fab fa-twitter"></i></a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav">English</a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav">Mypage</a>
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav">Logout</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="cont-navbar">
+        <div class="navbar-wrapper">
+          <a href="./index.html" class="link-page">
+            <figure class="logo-img-cont">
+              <img
+                src="./images/logo.png"
+                alt="logo"
+                class="logo-footer logo-navbar"
+              />
+              <div class="logo-text">
+                <h1 class="website-name logo-name">Earth</h1>
+                <h2 class="website-name logo-name">Planet</h2>
+              </div>
+            </figure>
+          </a>
+          <nav class="link-nav-cont">
+            <ul class="cont-list">
+              <li class="single-cont-list">
+                <a href="./about.html" class="link-nav-page">About</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#main" class="link-nav-page">Program</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#join" class="link-nav-page">Join</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#sponsor" class="link-nav-page">Sponsor</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#" class="link-nav-page">News</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#" class="link-nav-page action-camp">CC Campaign</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+
+    <section class="header-cont">
+      <h1 class="heading">&nbsp;</h1>
+      <div class="intro intro-about">
+        <h1 class="intro-title about-title">World Environment Summit 2021</h1>
+        <p class="intro-desc about-desc">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsum,
+          incidunt delectus optio illo ducimus vero recusandae minus doloremque
+          ex repellat deserunt ipsa cupiditate quos, dolore facere voluptas
+          facilis non reiciendis? Lorem ipsum dolor sit amet consectetur
+          adipisicing elit. Neque aperiam similique facere fugit consequatur
+          explicabo?
+        </p>
+        <p class="about-contact">
+          Please contact us per Email for any further questions about WE Global
+          Summit 2021!
+        </p>
+        <a href="#" class="about-contact-link">
+          @ Lorem, ipsum dolor sit amet consectetur adipisicing.
+        </a>
+      </div>
+    </section>
```

Comment:
- [ ] Please adjust this section to match the original font, color, size, margin/padding

Yours:
![Screenshot from 2021-10-15 19-33-38](https://user-images.githubusercontent.com/62705947/137529468-39291bc0-4c89-486a-9cf3-9fc0346e8c90.png)
Original:
![Screenshot from 2021-10-15 19-33-46](https://user-images.githubusercontent.com/62705947/137529483-1790ba1c-8cc3-4834-895e-386a6a418b0e.png)